DBlock Game/DBlock Game/Program.cs

```diff
@@ -1558,6 +1563,239 @@ public static void loading()
+                {
+                    case "1":
+                        chance = 5;
+                        enemyMovement(1);
+                        surviveCheck(playerLocation,enemyLocation);
+                        chance = 2;
+                        westHallway_Basement();
+                        break;
+                    case "2":
+                        chance = 5;
+                        enemyMovement(1);
+                        surviveCheck(playerLocation, enemyLocation);
+                        chance = 2;
+                        westHallway_Basement();
+                        break;
+
+                    case "3":
+                        enemyMovement(2);
+                        surviveCheck(playerLocation, enemyLocation);
+                        northHallway_Basement();
+                        break;
+
+                    case "4":
+                        enemyMovement(2);
+                        surviveCheck(playerLocation,enemyLocation);
+                        southHallway_Basement();
+                        break;
+
+                    case "map":
+                        Map(2);
+                        break;
+
+                    case "inv":
+                        Inventory();
+                        break;
+
+                    case "help":
+                        Help();
+                        break;
+
+                    default:
+                        fail = true;
+                        break;
+                } 
+            } while (fail == true);
+        }
+        /**
+        * template for the hallways jus need rooms attached to the case statements
+        */
+        public void northHallway_Basement()
+        {
+            loading();
+            TextBox("You have entered the north hall way", Player, false);
+            TextBox(enemyInfo, Player, false);
+            TextBox("You see a couple of bins and a table that you could possibly hide behind", Player, false);
+            TextBox("How would you like to procede??", Player, true);
+            bool fail = false;
+            do
+            {
+                TextBox("What should I do now? \n" + "║1: behind bins  \n║2: under table  \n║3:  \n║4: Move to West Hallway", Player, true);
+                string temp = Console.ReadLine();
+                switch (temp)
+                {
+                    case "1":
+                        chance = 5;
+                        enemyMovement(1);
+                        surviveCheck(playerLocation, enemyLocation);
+                        chance = 2;
+                        northHallway_Basement();
+                        break;
+
+                    case "2":
+                        chance = 5;
+                        enemyMovement(1);
+                        surviveCheck(playerLocation, enemyLocation);
+                        chance = 2;
+                        northHallway_Basement();
+                        break;
+
+                    case "3":
```

Comment:
if this is meant to be empty it shouldn't have a break

Reply:
i had left these empty for when i had a case to enter other rooms